src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -172,7 +172,7 @@ private void testAddInstructorAction() throws Exception {
         courseEditPage.clickCourseLevelPrivilegesLink(newInstructorIndex, 7);
         
         courseEditPage.clickAddSectionLevelPrivilegesLink(newInstructorIndex);
-        courseEditPage.clickSectionSelectionCheckBox(newInstructorIndex, 0, 0);
+        courseEditPage.clickSectionSelectionCheckBox(newInstructorIndex, 0, 1);
```

Comment:
Changing this to `1` would cause the order (1, 0, 2) for section panels in the previous implementation, but would remain as (0, 1, 2) under the change in this PR, thereby testing for section panel order under an existing test. This also explains the minimal change to the test results.